### ISSUE TITLE
nav links to /!import%20url resulting in 404 warning

### ISSUE THREAD

**Issue by fjp** [jdoiro3/mkdocs-multirepo-plugin#5]
Thanks for the plugin. I am trying to set it up with the following parent `mkdocs.yml`:

```
site_name: Documentation

use_directory_urls: false # Either true or false results in the same issue

plugins:
  - search
  - multirepo

nav:
  - Main: index.md
  - DemoRepo: '!import https://github.com/jdoiro3/mkdocs-multirepo-demoRepo1?branch=main'
```

This builds/serves fine and I can see a temporary folder being created where demo repo is cloned. However, when I try to navigate to the "DemoRepo" menu item I get a 404 page because the link is:

> http://127.0.0.1:8000/!import%20https:/github.com/jdoiro3/mkdocs-multirepo-demoRepo1?branch=main

I also see a warning:

> WARNING  -  [15:35:39] "GET /!import%20https:/github.com/jdoiro3/mkdocs-multirepo-demoRepo1?branch=main HTTP/1.1" code 404

The python version I am using is 3.9.10 on Windows and the following mkdocs versions:

```
mkdocs                     1.2.3
mkdocs-material            8.1.8
mkdocs-material-extensions 1.0.3
mkdocs-multirepo-plugin    0.2.6
```

Do you have any idea what might cause this issue?

**Comment by jdoiro3** [comment]
Ubuntu but just tried the same config on my Windows machine and didn't reproduce the bug. My Windows machine is using Python 3.9.2. Below is my *venv* setup.

```text
asyncio==3.4.3
click==8.0.3
colorama==0.4.4
ghp-import==2.0.2
importlib-metadata==4.11.1
Jinja2==3.0.3
Markdown==3.3.6
MarkupSafe==2.1.0
mergedeep==1.3.4
mkdocs==1.2.3
mkdocs-multirepo-plugin==0.2.6
packaging==21.3
pyparsing==3.0.7
python-dateutil==2.8.2
PyYAML==6.0
pyyaml-env-tag==0.1
six==1.16.0
tqdm==4.62.3
watchdog==2.1.6
zipp==3.7.0
```
I really want to figure out what's going on here haha.

**Comment by fjp** [comment]
I just tried again with the config I posted and it works. I was using another config all the time and didn't reported it properly in my first post. Sorry about that. 

The issue can be reproduced (I think) if you have another nav item after an imported one. For me the following nav config leads to the issue:

```
nav:
  - Main: index.md
  - DemoRepo: '!import https://github.com/jdoiro3/mkdocs-multirepo-demoRepo1?branch=main'
  - Main2: index.md
```

The DemoRepo url links to `http://127.0.0.1:8000/!import%20https:/github.com/jdoiro3/mkdocs-multirepo-demoRepo1?branch=main` and "Main2" links to the imported index.md of the https://github.com/jdoiro3/mkdocs-multirepo-demoRepo1, which I expected to link to the index.md of the parent repo.

**Comment by fjp** [comment]
The issue seems to be here on line 105:

https://github.com/jdoiro3/mkdocs-multirepo-plugin/blob/0aefb6f5803aae0c9b8d1bb43f5d71c7bbaa7c5f/mkdocs_multirepo_plugin/plugin.py#L100-L107

**Comment by jdoiro3** [comment]
Thank you for looking into this and helping out. You were correct. It was a bug I introduced when adding asynchronous importing to the plugin. It should be fixed in `v0.2.7`. I still need to add additional testing for these cases.

**Comment by fjp** [comment]
Awesome, thank you for the quick handling of this issue! I just tried `v0.2.7` and it seems to work as expected. So from my side you can close this issue. Thanks again for this handy plugin.
